cheatsheets/Ruby_on_Rails_Cheatsheet.md

```diff
@@ -20,7 +20,7 @@ open("\| os command here")
 
 While the power of these commands is quite useful, extreme care should be taken when using them in a Rails based application. Usually, its just a bad idea. If need be, a whitelist of possible values should be used and any input should be validated as thoroughly as possible.
 
-The Ruby Security Reviewer's Guide has a [section on injection](http://code.google.com/p/ruby-security/wiki/Guide#Good_ol%27_shell_injection) and there are a number of OWASP references for it, starting at the top: [Command Injection](https://www.owasp.org/index.php/Command_Injection).
+Check out the guides from [Rails](https://guides.rubyonrails.org/security.html#command-line-injection) and [OWASP](https://www.owasp.org/index.php/Command_Injection) for more on command injection.
```

Comment:
The tone of this is a bit informal compared to the rest of the cheat sheet. Perhaps something more like:

> The guides from [Rails] and [OWASP] contain further information on...

Reply:
Agree, updated

Reply:
Looks good - thanks.